evennia/utils/tests/test_eveditor.py

```diff
@@ -168,13 +168,13 @@ def test_eveditor_COLON_UU(self):
             eveditor.CmdLineInput(),
             'First test "line".',
             raw_string='First test "line".',
-            msg='01First test "line" .',
+            msg='01First test "line".',
         )
         self.call(
             eveditor.CmdLineInput(),
             "Second 'line'.",
             raw_string="Second 'line'.",
-            msg="02Second 'line' .",
+            msg="02Second 'line'.",
```

Comment:
It seems like a good opportunity to actually add a new test here to verify the echo is indeed as expected?